perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -2705,7 +2707,7 @@ private ExtSource getGroupExtSourceForSynchronization(PerunSession sess, Group g
 		if(overwriteUserAttributes != null && !overwriteUserAttributes.isEmpty()) {
 			//remove all white spaces and invisible characters
 			overwriteUserAttributes = overwriteUserAttributes.replaceAll("\\s", "");
-			overwriteUserAttributesList = Arrays.asList(overwriteUserAttributes.split(","));
+			overwriteUserAttributesList = asList(overwriteUserAttributes.split(","));
```

Comment:
What is the reason for this? I really prefer Arrays.asList, because it is much easier for human to read.

Reply:
Fixed